app/header/header.css

```diff
@@ -0,0 +1,188 @@
+a {
+  color: #374551;
+  font-size: 18px;
+}
+header {
+  min-height: 67px;
+}
+.navbar {
+  border-bottom: 1px solid #edf0f5;
+  background-color: #fff;
+  min-height: 67px;
+}
+.nav > li:focus > a, .nav > li:hover > a {
+  background: #ffc300;
+  color: #fff;
+  border-radius: 2px;
+}
+.navbar-nav > li {
```

Comment:
could we use post-css nested style to have everything more compact and easier to maintain  ? @rychkog  what do you think ?